AuxTel/Standard-Operations/Daytime-Operations/latiss-daily-calibrations-BIAS-DARK-FLAT-all-filters-empty-procedure.rst

```diff
@@ -43,232 +45,246 @@ Procedure Steps
+        exp_times_dark: 6
+        n_flat: 20
+        exp_times_flat: 30
+        script_mode: BIAS_DARK_FLAT
+        filter: empty_1
+        grating: SDSSy_65mm
+
+    5. **: Set configuration for empty_1.**
+
+    .. code-block:: text
+      :caption: :file:`auxtel/make_latiss_calibrations.py`
+
+        n_bias: 3
+        n_dark: 3
+        exp_times_dark: 1
+        n_flat: 20
+        exp_times_flat: 1
+        script_mode: BIAS_DARK_FLAT
+        filter: empty_1
+        grating: empty_1
+
+    6. **: Set sequence for the daily Photon Transfer Curve (PTC). Skip it if the previous sets were queued manually.**
+
+    .. code-block:: text
+      :caption: :file:`auxtel/take_image_latiss.py`
+
+        image_type: FLAT
+        filter: SDSSr_65mm
+        grating: empty_1
+        reason: daily_PTC
+        exp_times:
+                0.25,
+                0.25,
+                1.42,
+                1.42,
+                6.53,
+                6.53,
+                4.23,
+                4.23,
+                30.04,
+                30.04,
+                12.56,
+                12.56,
+                57.75,
+                57.75,
+                8.13,
+                8.13,
+                2.73,
+                2.73,
+                3.40,
+                3.40,
+                1.77,
+                1.77,
+                111.03,
+                111.03,
+                37.35,
+                37.35,
+                0.48,
+                0.48,
+                0.59,
+                0.59,
+                10.10,
+                10.10,
+                1.14,
+                1.14,
+                0.20,
+                0.20,
+                89.29,
+                89.29,
+                71.81,
+                71.81,
+                0.38,
+                0.38,
+                0.31,
+                0.31,
+                19.43,
+                19.43,
+                2.20,
+                2.20,
+                15.62,
+                15.62,
+                0.92,
+                0.92,
+                0.74,
+                0.74,
+                24.16,
+                24.16,
+                5.25,
+                5.25,
+                46.44,
+                46.44
+
+
+    7. **: Set sequence for daily_flats.**
+
+    .. code-block:: text
+      :caption: :file:`auxtel/take_image_latiss.py`
+
+        image_type: FLAT
+        filter: SDSSr_65mm
+        grating: empty_1
+        reason: daily_sflat,
+        exp_times:
+                        0.5,
+                        0.5,
+                        0.5,
+                        0.5,
+                        0.5,
+                        12.8,
+                        12.8,
+                        12.8,
+                        12.8,
+                        12.8
 
-.. code-block:: text
-  :caption: :file:`auxtel/calibrations/power_on_atcalsys.py`
 
+#. Run the script :file:`add_block.py` to the ATQueue  with the following configuration:
```

Comment:
Also here, describe the BLOCK in a couple of sentences. And below you can just describe the scripts.